Lib/test/test_unittest/testmock/testmock.py

```diff
@@ -115,6 +115,26 @@ def f(): pass
         with self.assertRaises(TypeError):
             mock()
 
+    def test_create_autospec_should_be_configurable_by_kwargs(self):
+        """If kwargs are given to set side_effect & return_value, the function
+        must configure the parent mock during initialization."""
+        class Result:
+            def get_result(self):
+                pass
+
+        mocked_result = 'mocked value'
+        class_mock = create_autospec(spec=Result, **{
+            # Test both side_effect & return_value which is why the 2nd
+            # parameter of side_effect is DEFAULT.
+            'return_value.get_result.side_effect': [ValueError, DEFAULT],
+            'return_value.get_result.return_value': mocked_result})
```

Comment:
Where else is this style of "string with docs indicate traverse through attributes" used?

Reply:
Removed noisy comments.